tags/tag-workloads-foundation/charter.md

```diff
@@ -1 +1,57 @@
-Charter content here
\ No newline at end of file
+# CNCF TAG Workloads Foundation
+
+## Mission
+
+To define and advance practices and standards for fundamental cloud native workload execution environments and their related lifecycle management within cloud native systems, applications, and architectures. This supports the CNCF's technical vision by addressing critical problems faced by adopters and contributing to a robust cloud native ecosystem.
+
+## Scope
+
+This TAG covers the following sub-domains and topics within its scope:
+- Containers
+- OS
+- Runtime
+- Virtual Machines
+- Serverless
+- Web Assembly
+- Batch
+- Serving
+- Scheduling
+- Orchestrator
+- Deployment
+- Dynamic Scaling
+- CI/CD
+
+## Out of Scope
+
+- Anything not explicitly mentioned in the scope above is out of scope.
+- This TAG will collaborate with other TAGs on overlapping areas, but will not focus on topics explicitly owned by other TAGs and initiatives.
```

Comment:
```suggestion
```
I suggest removing this line. Since area overlap is inevitable, and each tag will eventually have a list to declare the projects it is responsible for, it would be difficult to clarify which tag it belongs to. It is possible that a project is owned by more than one tag.

Reply:
Removed.